contents/docs/self-host/runbook/clickhouse/sharding-and-replication.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Sharding and replication
+sidebar: Docs
+showTitle: true
+---
+
+
+Sharding is a horizontal cluster scaling strategy that puts parts of one ClickHouse database on different shards. This can help you:
```

Comment:
```suggestion
Sharding is a horizontal cluster scaling strategy that puts parts of one ClickHouse database on different shards. This can help you to:
```